2025-05-09 13:20:49,479 - __main__ - ERROR - Hiba a kalibráció betöltésekor innen: C:Users\Nándi\Desktop\Python_programok\Onlab\calibration.json: [Errno 2] No such file or directory: 'C:Users\\Nándi\\Desktop\\Python_programok\\Onlab\\calibration.json'
2025-05-09 13:20:49,479 - __main__ - INFO - Alapértelmezett kalibrációs értékek használata
2025-05-09 13:20:49,479 - __main__ - INFO - Kapcsolódás a Dashboard szerverhez: 10.150.0.1:29999
2025-05-09 13:20:49,495 - __main__ - INFO - Kapcsolódva a Dashboard szerverhez
2025-05-09 13:20:49,495 - __main__ - INFO - Kapcsolódás a Másodlagos Interfészhez: 10.150.0.1:30002
2025-05-09 13:20:49,495 - __main__ - INFO - Kapcsolódás a robothoz: 10.150.0.1:30002...
2025-05-09 13:20:49,510 - __main__ - INFO - Sikeresen kapcsolódva a Másodlagos Interfészhez!
2025-05-09 13:20:49,510 - __main__ - INFO - Távvezérlési mód ellenőrzése...
2025-05-09 13:20:49,510 - __main__ - INFO - Robot mód lekérdezése...
2025-05-09 13:20:49,510 - __main__ - INFO - Robot mód: Robotmode: RUNNING
2025-05-09 13:20:49,510 - __main__ - INFO - Biztonsági állapot ellenőrzése...
2025-05-09 13:20:49,510 - __main__ - INFO - Biztonsági állapot: Safetystatus: NORMAL
2025-05-09 13:20:49,510 - __main__ - INFO - Sikeresen kapcsolódva a robothoz!
2025-05-09 13:20:54,476 - __main__ - INFO - Robot mód lekérdezése...
2025-05-09 13:20:54,476 - __main__ - INFO - Biztonsági állapot lekérdezése...
2025-05-09 13:20:54,476 - __main__ - INFO - Program állapot lekérdezése...
2025-05-09 13:20:54,476 - __main__ - INFO - Betöltött program lekérdezése...
2025-05-09 13:20:54,483 - __main__ - INFO - Robot állapotinformációk sikeresen lekérdezve
2025-05-09 13:21:14,547 - root - WARNING - The connection was lost to the robot. Please connect and try running again.
2025-05-09 13:21:19,996 - __main__ - INFO - Biztonsági mód kikapcsolva
2025-05-09 13:21:24,183 - __main__ - INFO - Biztonsági mód bekapcsolva
2025-05-09 13:21:59,738 - __main__ - ERROR - Hiba a kalibráció betöltésekor innen: C:Users\Nándi\Desktop\Python_programok\Onlab\calibration.json: [Errno 2] No such file or directory: 'C:Users\\Nándi\\Desktop\\Python_programok\\Onlab\\calibration.json'
2025-05-09 13:21:59,738 - __main__ - INFO - Alapértelmezett kalibrációs értékek használata
2025-05-09 13:21:59,738 - __main__ - INFO - Kapcsolódás a Dashboard szerverhez: 10.150.0.1:29999
2025-05-09 13:21:59,754 - __main__ - INFO - Kapcsolódva a Dashboard szerverhez
2025-05-09 13:21:59,754 - __main__ - INFO - Kapcsolódás a Másodlagos Interfészhez: 10.150.0.1:30002
2025-05-09 13:21:59,754 - __main__ - INFO - Kapcsolódás a robothoz: 10.150.0.1:30002...
2025-05-09 13:21:59,756 - __main__ - INFO - Sikeresen kapcsolódva a Másodlagos Interfészhez!
2025-05-09 13:21:59,756 - __main__ - INFO - Távvezérlési mód ellenőrzése...
2025-05-09 13:21:59,760 - __main__ - INFO - Robot mód lekérdezése...
2025-05-09 13:21:59,763 - __main__ - INFO - Robot mód: Robotmode: RUNNING
2025-05-09 13:21:59,763 - __main__ - INFO - Biztonsági állapot ellenőrzése...
2025-05-09 13:21:59,765 - __main__ - INFO - Biztonsági állapot: Safetystatus: NORMAL
2025-05-09 13:21:59,766 - __main__ - INFO - Sikeresen kapcsolódva a robothoz!
2025-05-09 13:22:04,809 - __main__ - INFO - Robot mód lekérdezése...
2025-05-09 13:22:04,824 - __main__ - INFO - Biztonsági állapot lekérdezése...
2025-05-09 13:22:04,824 - __main__ - INFO - Program állapot lekérdezése...
2025-05-09 13:22:04,824 - __main__ - INFO - Betöltött program lekérdezése...
2025-05-09 13:22:04,829 - __main__ - INFO - Robot állapotinformációk sikeresen lekérdezve
2025-05-09 13:23:18,767 - __main__ - INFO - Mozgás a kezdőpozícióba: [X: -37.0mm, Y: -295.0mm, Z: -10.0mm, Rx: 2.20, Ry: 2.20, Rz: 0.00]
2025-05-09 13:23:18,767 - __main__ - INFO - Toll felemelése a kezdőpozícióba való mozgás előtt...
2025-05-09 13:23:18,767 - __main__ - INFO - Toll felemelése...
2025-05-09 13:23:18,767 - __main__ - INFO - Toll felemelése a következő Z pozícióra: -134mm
2025-05-09 13:23:18,767 - __main__ - INFO - Script küldése: def pen_up():
              current_pose = get_actual_tcp_pose()
              current_pose[2] = -0.134
              movel(current_pose, a=0.5, v=0.1)
            end
            pen_up()
2025-05-09 13:23:20,781 - __main__ - INFO - Toll felemelve, új Z pozíció: -134mm
2025-05-09 13:23:20,781 - __main__ - INFO - TCP mozgatása a következő pozícióba: [X: -37.0mm, Y: -295.0mm, Z: -134.0mm, Rx: 2.20, Ry: 2.20, Rz: 0.00]
2025-05-09 13:23:20,781 - __main__ - INFO - Sebesség: 0.1, Szegmensek: 10, Rajzolás: Nem
2025-05-09 13:23:20,781 - __main__ - INFO - Biztonsági mód: Mozgás felosztása 10 szegmensre
2025-05-09 13:23:23,976 - __main__ - INFO - Mozgatási parancs küldése a 1. szegmenshez 10-ből: movel(p[-0.037000, -0.295000, -0.134000, 2.200000, 2.200000, 0.000000], a=0.5, v=0.1)
2025-05-09 13:23:23,976 - __main__ - INFO - Script küldése: movel(p[-0.037000, -0.295000, -0.134000, 2.200000, 2.200000, 0.000000], a=0.5, v=0.1)
2025-05-09 13:23:23,976 - __main__ - INFO - Várakozás a szegmens mozgásának befejezésére...
2025-05-09 13:23:25,987 - __main__ - INFO - A 1. szegmens befejezve, új pozíció: [X: -37.0mm, Y: -295.0mm, Z: -134.0mm, Rx: 2.20, Ry: 2.20, Rz: 0.00]
